stable/provazio/templates/controller-namespace-admin-clusterrole.yaml

```diff
@@ -1,11 +1,11 @@
-{{- if and (.Values.controller.enabled) (has "IguazioTenantAppServiceSet" .Values.controller.crd.kinds) }}
+{{- if and (.Values.controller.enabled) (has "IguazioTenant" .Values.controller.crd.kinds) }}
```

Comment:
it was done on purpose, because only iguazio-platform tenant should create the `ClusterRole` to be used by any new tenant using RoleBinding with reference to this cluster role